site/docs/v1/tech/tutorials/gate-accounts-until-verified.adoc

```diff
@@ -0,0 +1,131 @@
+
+Navigate to [breadcrumb]#Customizations -> Email Templates#. Edit the `Email Verification` template by clicking on the blue "Edit" button.
+
+Modify the HTML and text templates as desired and then save them. You can also localize the messages. 
+
+image::tutorials/gate-accounts-until-verified/email-template-customization.png[Customizing the email verification email template,width=1200]
+
+== Configure the Application
+
+Navigate to [breadcrumb]#Applications#. Edit your application by clicking on the blue "Edit" button. 
+
+Enable self registration by selecting the [breadcrumb]#Registration# tab, and then scrolling to the [breadcrumb]#Self service registration# section. 
+
+[NOTE]
+====
+You can create a user with an unverified email address in other ways than self service registration. However, self service registration is the simplest way to create a user without a verified email.
+====
+
+Enable self service registration and configure the displayed and required fields as desired.
+
+image::tutorials/gate-accounts-until-verified/application-enable-self-registration.png[Enabling application self registration,width=1200]
+
+Next, navigate to the [breadcrumb]#OAuth# tab. Enable the `Authorization Code` grant. 
+
+In addition, ensure you have entered a valid URL in the [field]#Authorized redirect URLs# field. (The value of this redirect URL doesn't really matter for this tutorial. But for production use, ensure the code at that endpoint can exchange an authorization code for a token.)
+
+Save the application configuration. You'll be returned to the list of applications.
+
+Click on the green "View" button of your application to see the details of your application.
+
+image::tutorials/gate-accounts-until-verified/view-application-details.png[View application details to see the relevant URLs,width=1200]
+
+Look for the [field]#Registration URL#. It'll be something like `https://local.fusionauth.io/oauth2/register?client_id=85a03867-dccf-4882-adde-1a79aeec50df&response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Foauth-redirect`. Copy that value as you'll use it below.
+
+At the same time, find the [field]#OAuth IdP login URL#, and copy that as well.
+
+== Test It Out
+
+Now, test out the functionality. Register a new account. Make sure to use a different browser or incognito mode so that the admin account you logged in with while configuring FusionAuth doesn't interfere with new user registration.
+
+Visit the registration URL copied above. You'll be prompted to register. Fill out the form with a valid email address. After registration is complete, you'll receive an email prompting you to verify the email address. 
+
+Here is the email in the mailcatcher user interface:
+
+image::tutorials/gate-accounts-until-verified/verification-email-mailcatcher.png[Verification email in mailcatcher,width=1200,role=bottom-cropped]
+
+After you have registered but before you have verified the email by clicking the link, you'll be gated. 
```

Comment:
I've read this line a few times and still think its sort of confusing.
Maybe something like:?
"You'll be gated if you have registered but have not yet verified the email by clicking the link."